examples/face_landmarker/android/app/src/main/java/com/google/mediapipe/examples/facelandmarker/fragment/FaceBlendshapesResultAdapter.kt

```diff
@@ -27,17 +27,23 @@ class FaceBlendshapesResultAdapter :
     RecyclerView.Adapter<FaceBlendshapesResultAdapter.ViewHolder>() {
     companion object {
         private const val NO_VALUE = "--"
+        const val TAG = "FaceBlendshapesResultAdapter"
     }
 
     private var categories: MutableList<Category?> = MutableList(52) { null }
 
     fun updateResults(faceLandmarkerResult: FaceLandmarkerResult? = null) {
         categories = MutableList(52) { null }
         if (faceLandmarkerResult != null) {
-            val sortedCategories = faceLandmarkerResult.faceBlendshapes().get()[0].sortedBy { -it.score() }
-            val min = kotlin.math.min(sortedCategories.size, categories.size)
-            for (i in 0 until min) {
-                categories[i] = sortedCategories[i]
+            try {
```

Comment:
Rather than using a generic try/catch here (which can potentially obstruct other errors that get thrown), can we just add another check to make sure faceBlendshapes() isn't empty?